app/test_helpers.go

```diff
@@ -236,31 +236,33 @@ func IntegrationTestNetworkConfig() network.Config {
 	// Start with the default genesis state
 	appGenState := ModuleBasics.DefaultGenesis(encCfg.Codec)
 
-	// Default genesis doesn't have tokens in Token registry, so here we add one:
+	// Override default leverage registry with one more suitable for testing
 	var leverageGenState leveragetypes.GenesisState
 	if err := cdc.UnmarshalJSON(appGenState[leveragetypes.ModuleName], &leverageGenState); err != nil {
 		panic(err)
 	}
-	leverageGenState.Registry = append(leverageGenState.Registry, leveragetypes.Token{
-		BaseDenom:              params.BondDenom,
-		SymbolDenom:            params.DisplayDenom,
-		Exponent:               6,
-		ReserveFactor:          sdk.MustNewDecFromStr("0.1"),
-		CollateralWeight:       sdk.MustNewDecFromStr("0.05"),
-		LiquidationThreshold:   sdk.MustNewDecFromStr("0.05"),
-		BaseBorrowRate:         sdk.MustNewDecFromStr("0.02"),
-		KinkBorrowRate:         sdk.MustNewDecFromStr("0.2"),
-		MaxBorrowRate:          sdk.MustNewDecFromStr("1.5"),
-		KinkUtilization:        sdk.MustNewDecFromStr("0.2"),
-		LiquidationIncentive:   sdk.MustNewDecFromStr("0.18"),
-		EnableMsgSupply:        true,
-		EnableMsgBorrow:        true,
-		Blacklist:              false,
-		MaxCollateralShare:     sdk.MustNewDecFromStr("1"),
-		MaxSupplyUtilization:   sdk.MustNewDecFromStr("1"),
-		MinCollateralLiquidity: sdk.MustNewDecFromStr("0"),
-		MaxSupply:              sdk.NewInt(100000000000),
-	})
+	leverageGenState.Registry = []leveragetypes.Token{
+		{
+			BaseDenom:              params.BondDenom,
+			SymbolDenom:            params.DisplayDenom,
+			Exponent:               6,
+			ReserveFactor:          sdk.MustNewDecFromStr("0.1"),
+			CollateralWeight:       sdk.MustNewDecFromStr("0.05"),
+			LiquidationThreshold:   sdk.MustNewDecFromStr("0.05"),
+			BaseBorrowRate:         sdk.MustNewDecFromStr("0.02"),
+			KinkBorrowRate:         sdk.MustNewDecFromStr("0.2"),
+			MaxBorrowRate:          sdk.MustNewDecFromStr("1.5"),
+			KinkUtilization:        sdk.MustNewDecFromStr("0.2"),
+			LiquidationIncentive:   sdk.MustNewDecFromStr("0.18"),
+			EnableMsgSupply:        true,
+			EnableMsgBorrow:        true,
+			Blacklist:              false,
+			MaxCollateralShare:     sdk.MustNewDecFromStr("1"),
+			MaxSupplyUtilization:   sdk.MustNewDecFromStr("1"),
+			MinCollateralLiquidity: sdk.MustNewDecFromStr("0"),
+			MaxSupply:              sdk.NewInt(100000000000),
+		},
+	}
```

Comment:
we need to Atom as well.

Reply:
This isn't the default registry, it's one that allows us to run CLI tests with UMEE only

Reply:
I know, but don't we need it for tests?

Reply:
Switching to default registry here would cause a few downstream tests to fail (they're expecting these specific values) but we can do that at a later date. Created #1414